### ISSUE TITLE
Additional Options for Jobs\Schedules

### ISSUE THREAD

**Issue by adamdriscoll** [ironmansoftware/universal-automation#48]
We should probably consider putting some of these options from Task Scheduler in UA. 

![image](https://user-images.githubusercontent.com/1256531/70807952-dfbb5300-1d7b-11ea-8040-26c29537e128.png)

**Comment by leeberg** [comment]
I'm thinking these would bind to the Schedule - like a "Schedule Params"

Not Bound to Script/Job.

This way One script could have a variety of scheduled executions each with unique behavior

**Comment by adamdriscoll** [comment]
Yep. Totally agree. These should be on the New\Set-UASchedule cmdlet.
